ci/jenkins/test.sh

```diff
@@ -295,6 +302,13 @@ function wait_for_antrea_windows_processes_ready {
         done
         sleep 10
     done
+    WIN_BRINT_DHCP=$(ssh -o StrictHostKeyChecking=no -n administrator@${WINIP} "powershell.exe (Get-NetIPInterface -InterfaceAlias br-int -AddressFamily IPv4).Dhcp")
+    if [[ $WIN_DHCP == $WIN_BRINT_DHCP ]]; then
+        echo "New created uplink DHCP status is consistent with the original adapter."
+    else
+        echo "New created uplink DHCP status is different with the original adapter."
+        exit 1
+    fi
```

Comment:
So it is not an e2e case, but a "must have" check in CI setup?

Reply:
Since we need to clean host environment to get the original interface DHCP status, I feel this is more related to CI infrastructure not e2e case, and it would be better to added this step to agent status check in CI pipeline.